### PR TITLE
Cosmic Bedsheet

### PR DESCRIPTION
**Pull request by ghost**
Adds cosmic space bedsheet to the code from Oracle
Need some ideas on where to code/map in this bad boy, currently you can get it on pubby and the clown shuttle since those have random bedsheets mapped in

[why]: Because it looks sweet

### CODE REVIEW

**Comment by Buggy123** [comment]
Missing tags;
Priority: CRITICAL
Test Merge Candidate

File: code/game/objects/structures/bedsheet_bin.dm [hunk 1]
```diff
@@ -223,6 +223,15 @@ LINEN BINS
 	item_color = "ian"
 	dream_messages = list("a dog", "a corgi", "woof", "bark", "arf")
 
+/obj/item/bedsheet/cosmos
+	name = "cosmic space bedsheet"
+	desc = "Made from the dreams of those who wonder at the stars."
+	icon_state = "sheetcosmos"
+	item_color = "cosmos"
+	dream_messages = list("the infinite cosmos", "Hans Zimmer music", "a flight through space", "the galaxy", "being fabulous", "shooting stars")
+	luminosity = 1
```
Comment (Cyberboss) [inline code/game/objects/structures/bedsheet_bin.dm]: This doesn't do what you think it does, remove it

**Review by Cyberboss — DISMISSED** [review]

**Comment by epochayur** [comment]
could it be an arcade machine prize?

**Comment by Spaceinaba** [comment]
should be craftable by washing a bedsheet with a bluespace crystal

**Comment by RandomMarine** [comment]
@Spaceinaba That but it also warps the washer and sheet somewhere and you have to go find it.

**Comment by optimumtact** [comment]
in game shot please

**Pull request merged**